File: Flix.Backend/Flix.Services/Database/ActivityType.cs
namespace Flix.Services.Database
{
    /// <summary>
    /// Type of activity recorded in a user's activity stream. Drives the
    /// "Recent Activity" admin widget and the friends/following feed.
    /// </summary>
    public enum ActivityType
    {
        JoinedPlatform = 0,
        WatchedMovie = 1,
        ReviewedMovie = 2,
        LikedMovie = 3,
        AddedToWatchlist = 4,
        CreatedList = 5,
        FollowedUser = 6,
        RequestedMovie = 7,
        JoinedClash = 8,
        WonClash = 9
    }
}

File: Flix.UI/flix_desktop/lib/enums/activity_type.dart
import 'package:json_annotation/json_annotation.dart';

enum ActivityType {
  @JsonValue(0)
  joinedPlatform,
  @JsonValue(1)
  watchedMovie,
  @JsonValue(2)
  reviewedMovie,
  @JsonValue(3)
  likedMovie,
  @JsonValue(4)
  addedToWatchlist,
  @JsonValue(5)
  createdList,
  @JsonValue(6)
  followedUser,
  @JsonValue(7)
  requestedMovie,
  @JsonValue(8)
  joinedClash,
  @JsonValue(9)
  wonClash,
  @JsonValue(10)
  votedOnClash
}
